scipy/signal/tests/test_spectral.py

```diff
@@ -9,10 +9,15 @@
 
 from scipy import signal
 from scipy.fft import fftfreq
-from scipy.signal import (periodogram, welch, lombscargle, csd, coherence,
-                          spectrogram, stft, istft, check_COLA, check_NOLA)
+from scipy.signal import (periodogram, welch, lombscargle, coherence,
+                          spectrogram, check_COLA, check_NOLA)
 from scipy.signal._spectral_py import _spectral_helper
 
+# Compare ShortTimeFFT.stft() / ShortTimeFFT.stft() with stft() / istft():
+from ._scipy_spectral_test_shim import stft_compare as stft
+from ._scipy_spectral_test_shim import istft_compare as istft
+from ._scipy_spectral_test_shim import csd_compare as csd
```

Comment:
Instead of doing this, can we add a new `test_spectral_stft.py` with contents:
```
# Duplicate test_spectral tests, but using newer (I)STFT functions
from test_spectral import *
from ._scipy_spectral_test_shim import stft_compare as stft
from ._scipy_spectral_test_shim import istft_compare as istft
from ._scipy_spectral_test_shim import csd_compare as csd
```
? I *think* that would make it so that both types of tests run now and in the future.

Reply:
Also let's use absolute imports please.

Reply:
I already ported all relevant tests from `test_spectral.py` to `test_short_time_fft.py` by hand. The original purpose of the test shims was to have an easy way of cross-checking during implementing. In my opinion, the test shims do not add any benefit to unit testing (anymore). I left them in, since they nicely document the more subtle differences of the implementation choices. I would be fine with removing them.